docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,734 @@
+
+            from qiskit.quantum_info import SparsePauliOp, Pauli
+
+            qubit_op = SparsePauliOp(Pauli("XYZY")), coeff=-6j)
+
+     -
+
+
+Converters
+----------
+
+The role of this sub-module was to convert the operators into other opflow operator classes (:class:`~qiskit.opflow.TwoQubitReduction`, :class:`~qiskit.opflow.PauliBasisChange`...).
+In the case of the :class:`~qiskit.opflow.CircuitSampler`, it traversed an operator and outputted approximations of its state functions using a quantum backend.
+Notably, this functionality has been replaced by the :mod:`~qiskit.primitives`.
+
+Circuit Sampler
+~~~~~~~~~~~~~~~
+
+.. list-table:: Migration of ``qiskit.opflow.CircuitSampler``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.CircuitSampler`
+     - :class:`~primitives.Estimator`
+     -
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.opflow import X, Z, StateFn, CircuitStateFn, CircuitSampler
+            from qiskit.providers.aer import AerSimulator
+
+            qc = QuantumCircuit(1)
+            qc.h(0)
+            state = CircuitStateFn(qc)
+            hamiltonian = X + Z
+
+            expr = StateFn(hamiltonian, is_measurement=True).compose(state)
+            backend = AerSimulator()
+            sampler = CircuitSampler(backend)
+            expectation = sampler.convert(expr)
+            expectation_value = expectation.eval().real
+
+     -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.primitives import Estimator
+            from qiskit.quantum_info import SparsePauliOp
+
+            state = QuantumCircuit(1)
+            state.h(0)
+            hamiltonian = SparsePauliOp.from_list([('X', 1), ('Z',1)])
+
+            estimator = Estimator()
+            expectation_value = estimator.run(state, hamiltonian).result().values
+
+     -
+
+Two Qubit Reduction
+~~~~~~~~~~~~~~~~~~~~
+.. list-table:: Migration of ``qiskit.opflow.TwoQubitReduction``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * -  :class:`~qiskit.opflow.TwoQubitReduction`
+     -  No direct replacement.
+     -  This class implements a chemistry-specific reduction for the ``ParityMapper`` class in ``qiskit-nature``.
+        The general symmetry logic this mapper depends on has been refactored to other classes in :mod:`~qiskit.quantum_info`,
+        so this specific :mod:`~qiskit.opflow` implementation is no longer necessary.
+
+Other Converters
+~~~~~~~~~~~~~~~~~
+
+.. list-table:: Migration of ``qiskit.opflow.converters``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.AbelianGrouper`
+     - No direct replacement.
+     - This class allowed a sum a of Pauli operators to be grouped. These type of groupings are now left to the primitives to handle (??).
```

Comment:
> These type of groupings are now left to the primitives to handle (??).

Yes, the given an operator which is a sum of Paulis then this is now the responsibilty of an Estimator - Aer Estimator has just added the support - the Runtime Estimator I believe does this as does the BackendEstimator (for info, without it a circuit is run for each Pauli, with it one for each group which can be way less circuits run so better performance)